src/undo.h

```diff
@@ -97,6 +97,8 @@ class CTxUndo
     uint256 refTx;            //hash of coins from ceased sidechains. It's not needed for ordinary coins and certs
     unsigned int firstBwtPos; //position of the first bwt.
 
+    CTxUndo(): vprevout(), refTx(), firstBwtPos(0) {};
```

Comment:
Is 0 a ggo default value for firstBwtPos? What if we have no bwt and no change?

Reply:
Current code actually fills the CTxUndo::firstBwtPos only for certificates which have bwt. Certificates with no bwts do not create a CTxUndo.
A way of structuring this code which is more coherent with the rest of the codebase is to type firstBwtPos as int and use -1 as default value

Reply:
In case there are no bwt, the CTxUndos for the ceased bwts are not generated at all in HandleCeasingScs nor restored in RevertCeasingScs
However, for clarity sake, I'll set -1 as default value for firstBwtPos (changing it to int) and skip the relevant processing on RevertCeasingScs should firstBwtPos  == -1